web/packages/teleterm/src/ui/appContext.ts

```diff
@@ -159,5 +167,14 @@ export default class AppContext implements IAppContext {
         request as SendNotificationRequest
       );
     });
+
+    this.subscribeToTshdEvent(
+      'sendPendingHeadlessAuthentication',
+      ({ request }) => {
+        this.headlessAuthenticationService.sendPendingHeadlessAuthentication(
```

Comment:
```suggestion
        return this.headlessAuthenticationService.sendPendingHeadlessAuthentication(
```

Otherwise the listener we add here returns immediately, making it possible to send multiple headless auth prompts to the Electron app from tshd.